APK Size Comparison is no longer working

https://github.com/ankidroid/Anki-Android/actions/runs/24627068099

> make sure you have checked out the target repository

```

2026-04-19T10:35:05.6700996Z Merged default JDK locations into /home/runner/.m2/toolchains.xml
2026-04-19T10:35:05.6705275Z Basic Caching: This build uses the open-source caching provider for reliable, path-based caching of Gradle dependencies. Upgrade available: for faster builds and advanced features, consider switching to the Enhanced Caching provider. See https://github.com/gradle/actions/blob/main/DISTRIBUTION.md for details.
2026-04-19T10:35:05.6795624Z ##[error]Error: No file in /home/runner/work/Anki-Android/Anki-Android matched to [**/*.gradle*,**/gradle-wrapper.properties,buildSrc/**/Versions.kt,buildSrc/**/Dependencies.kt,gradle/*.versions.toml,**/versions.properties], make sure you have checked out the target repository
2026-04-19T10:35:05.6826583Z Error: No file in /home/runner/work/Anki-Android/Anki-Android matched to [**/*.gradle*,**/gradle-wrapper.properties,buildSrc/**/Versions.kt,buildSrc/**/Dependencies.kt,gradle/*.versions.toml,**/versions.properties], make sure you have checked out the target repository
2026-04-19T10:35:05.6829093Z     at Qct (file:///home/runner/work/_actions/gradle/actions/v6/dist/setup-gradle/main/index.js:250:17)
2026-04-19T10:35:05.6830033Z     at async AD.restore (file:///home/runner/work/_actions/gradle/actions/v6/dist/setup-gradle/main/index.js:249:10844)
2026-04-19T10:35:05.6830833Z     at async cD.restore (file:///home/runner/work/_actions/gradle/actions/v6/dist/setup-gradle/main/index.js:263:236)
2026-04-19T10:35:05.6831626Z     at async VNe (file:///home/runner/work/_actions/gradle/actions/v6/dist/setup-gradle/main/index.js:294:388)
2026-04-19T10:35:05.6832302Z     at async gqt (file:///home/runner/work/_actions/gradle/actions/v6/dist/setup-gradle/main/index.js:336:695)
```

--- a/.github/workflows/compare_apk_size.yml
+++ b/.github/workflows/compare_apk_size.yml
@@ -42,6 +42,12 @@ jobs:
           echo y | keytool -genkeypair -dname "cn=AnkiDroid, ou=ankidroid, o=AnkiDroid, c=US" -alias $KEYALIAS -keypass $KEYPWD -keystore $KEYSTOREPATH -storepass $KEYSTOREPWD -keyalg RSA -validity 20000
         shell: bash
 
+      - name: Checkout PR
+        uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+          ref: 'refs/pull/${{ github.event.inputs.prNumber }}/head'
+
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v6
         timeout-minutes: 5
@@ -49,12 +55,6 @@ jobs:
           # Use open source provider: https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#basic-caching
           cache-provider: basic
           cache-read-only: true
-
-      - name: Checkout PR
-        uses: actions/checkout@v6
-        with:
-          fetch-depth: 0
-          ref: 'refs/pull/${{ github.event.inputs.prNumber }}/head'
 
       - name: Assemble PR APK
         # This makes sure we fetch gradle network resources with a retry

